graphql/resolvers.go

```diff
@@ -1002,13 +998,36 @@ func (r *queryResolver) Patch(ctx context.Context, id string) (*restModel.APIPat
 	if utility.StringSliceContains(statuses, evergreen.TaskAborted) {
 		if len(utility.StringSliceIntersection(statuses, evergreen.TaskFailureStatuses)) == 0 {
 			abortedStatus := evergreen.TaskAborted
-			patch.Status = &abortedStatus
+			status = &abortedStatus
```

Comment:
nit: can use utility.ToStringPtr to do this in one line rather than assigning TaskAborted to a temporary variable